dashboard/test/ui/features/popoutdrag.feature

```diff
@@ -1,7 +1,7 @@
 Feature: Blocks can be dragged from popouts
 
 Background:
-  Given I am on "http://learn.code.org/s/1/level/59?noautoplay=true"
+  Given I am on "http://studio.code.org/s/20-hour/stage/11/puzzle/1?noautoplay=true"
```

Comment:
Oops, I thought I left them all as learn... I'm just going to fix the test
runner so that it works with both learn and studio since we use studio now

On Thu, May 7, 2015 at 10:02 AM, Josh Lory <notifications@github.com> wrote:

> In dashboard/test/ui/features/popoutdrag.feature
> <https://github.com/code-dot-org/code-dot-org/pull/2331#discussion_r29872350>
> :
>
> > @@ -1,7 +1,7 @@
> >  Feature: Blocks can be dragged from popouts
> >
> >  Background:
> > -  Given I am on "http://learn.code.org/s/1/level/59?noautoplay=true"
> > +  Given I am on "http://studio.code.org/s/20-hour/stage/11/puzzle/1?noautoplay=true"
>
> Studio?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/code-dot-org/code-dot-org/pull/2331/files#r29872350>.
>